src/controller/adminController.js

```diff
@@ -169,6 +173,68 @@ const login = async function (req, res) {
     }
 }
 
+const updateAdmin = async function (req, res) {
```

Comment:
this api is not working propperly so please dont merge the code it is nedded to get fixed first